public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -171,7 +177,7 @@ export const AlertRuleForm: FC<Props> = ({ existing, prefill }) => {
               disabled={submitState.loading}
               type="button"
               fill="outline"
-              onClick={() => logInfo(LogMessages.cancelSavingAlertRule)}
+              onClick={cancelRuleCreation}
```

Comment:
Does it get triggered when the user closes the form with an existing rule?

Reply:
That's a great catch, same thing was happening with the creation of the rule. Fixed it.